README.org

```diff
@@ -46,6 +46,8 @@ If you use ~use-package~, here is the recipe for installing it.
     :bind (:map dired-mode-map ("C-o" . 'casual-dired-tmenu)))
 #+end_src
 
+Beyond ~C-o~ (or a binding of your choosing), you may want to bind ~s~ to ~casual-dired-sort-by-tmenu~: this will show the sorting menu when you press ~s~.
```

Comment:
Please replace this whole sentence with the following text:

```
In addition, you may wish to directly sort the Dired buffer by different criteria. Binding ~s~ (or one of your choosing) to ~casual-dired-sort-by-tmenu~ will accomplish this. Look below in the Configuration section to see how this can be done.
```